packages/edit-site/src/components/global-styles/header.js

```diff
@@ -12,7 +12,7 @@ import {
 import { isRTL, __ } from '@wordpress/i18n';
 import { chevronRight, chevronLeft } from '@wordpress/icons';
 
-function ScreenHeader( { title, description } ) {
+function ScreenHeader( { title, description, onClick } ) {
```

Comment:
Tiny nit: `onClick` makes me think that the entire header will be clickable. What if we named this prop `onBack` instead, since the prop is only for the back button?

We'd then pass `onBack` to the `NavigatorToParentButton` component's `onClick` prop.